client/schemas/blueprint-spec2-schema.json

```diff
@@ -500,13 +500,17 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
+                "parameter_name": {
```

Comment:
I would change the "_" to "-"
I know its nitpicking but I see its the syntax we went with here for the separator

Reply:
to be honest we have both underscores and hyphens, Shira set that in story.